packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,123 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <a
+        :class="[
+            $style['o-link'], {
+                [$style['o-link--bold']]: isBold,
+                [$style['o-link--noDecoration']]: !hasTextDecoration,
+                [$style['o-link--full']]: isFullWidth
+            }]"
+        :data-test-id="'link-component'"
```

Comment:
If this is a literal, I'd just make it a standard attribute, unless this has some weird interaction with where you're setting the `data-test-id` attribute in storybook?
```suggestion
        data-test-id="link-component"
```